src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -88,8 +101,17 @@ public UnleashContextProvider getContextProvider() {
         return contextProvider;
     }
 
+    public static void setRequestProperties(HttpURLConnection connection, UnleashConfig config) {
+        connection.setRequestProperty(UNLEASH_APP_NAME_HEADER, config.getAppName());
+        connection.setRequestProperty(UNLEASH_INSTANCE_ID_HEADER, config.getInstanceId());
+        for (String name : config.getCustomHttpHeaders().keySet()) {
```

Comment:
`maven-compiler-plugin` in the `pom.xml` specifies a source and target of 1.8, so I guess that makes the module Java 8. I'll change it to `forEach`, that is nicer – thanks!